Tasks/AzureCloudPowerShellDeploymentV1/Utility.ps1

```diff
@@ -77,6 +77,19 @@ function Get-AzureStoragePrimaryKey($storageAccount, [bool]$isArm)
     return $primaryStorageKey
 }
 
+function Get-SplitConfigStorageAccountName($publicConfigStorageAccountName)
+{
+    if(-not ([string]::IsNullOrEmpty($publicConfigStorageAccountName)) -and $publicConfigStorageAccountName.Contains("AccountKey"))
+    {
+        $publicConfig_StorageAccountName = $publicConfigStorageAccountName -split ";"
+        return $publicConfig_StorageAccountName[0]
```

Comment:
I think this is not enough to extract account name from connection string like this - DefaultEndpointsProtocol=https;AccountName=storagesample;AccountKey=<account-key>

Reply:
Our assumption from the Kusto logs that the string would be something like this 'hseydev;AccountKey=****'. To remove the AccountKey the above logic was used.